### PR TITLE
Yaml heads

### PR DESCRIPTION
**Pull request by mariaogryczak**
Add heads configuration to train_config.yml.

### CODE REVIEW

**Review by AgaMiko — APPROVED** [review]

**Review by wiktorfilipiuk — APPROVED** [review]

File: datasets/dataset.py [hunk 1]
```diff
@@ -128,8 +129,9 @@ def __init__(
         self,
         root_path: str,
         annotationfile_path: str,
-        classification_mode: str,
         is_pretraining: bool,
+        classification_mode: str,
```
Comment (majsylw) [inline datasets/dataset.py:133]: @mariaogryczak can we somehow delete this property?
I saw that this need to be set in PreTrainingModel, but here is totaly unused

File: datasets/dataset.py [hunk 2]
```diff
@@ -108,8 +108,9 @@ class VideoFrameDataset(torch.utils.data.Dataset):
         root_path: The root path in which video folders lie.
         annotationfile_path: The .txt annotation file containing
                              one row per video sample as described above.
-        classification_mode: mode for classification, choose from
-                             classification_mode.py
+        is_pretraining: flag to launch pretrianing block
+        classification_heads: dict with names of classification heads
+                              and number of their classes
```
Comment (majsylw) [inline datasets/dataset.py:113]: I added descriptions for these parameters

File: datasets/dataset.py [hunk 3]
```diff
@@ -46,13 +46,13 @@ class VideoRecord(object):
...
-    def __init__(self, row, root_datapath, num_classes_dict, landmarks_path=None):
+    def __init__(self, row, root_datapath, classification_heads):
         self._data = row
         self._path = os.path.join(root_datapath, row[0])
-        self.num_classes_dict = num_classes_dict
+        self.classification_heads = classification_heads
         try:
             with open(
-                os.path.join(landmarks_path, row[0] + "_properties.json"), "r"
+                os.path.join(self._path, row[0] + "_properties.json"), "r"
```
Comment (majsylw) [inline datasets/dataset.py:55]: Here I just fixed problem with landmarks and basic properties, now using time parameter should work with all datasets with HamNoSys

**Review by majsylw — APPROVED** [review]
@mariaogryczak @wiktorfilipiuk please, can you add some description about this yaml files in README.md in this PR?

**Comment by majsylw** [comment]
Also I think we need to move yaml files to some directory and then add it to .gitignore, becouse they can be change by each user independently

**Comment by mariaogryczak** [comment]
> Also I think we need to move yaml files to some directory and then add it to .gitignore, becouse they can be change by each user independently
Ok, but I don't know how to do it.

**Comment by majsylw** [comment]
> @majsylw Ok, but I don't know how to do it.

for now you just can add this two yaml files to .gitignore and it should be enough
For moving them sowhere (to special directory with name that will put it on the top of repository or bottom to easly navigate), you will need additionaly change paths in code (for assets)

**Pull request merged**